### PR TITLE
b2c all available stores are displayed in the grid

### PR DESCRIPTION
**Pull request by khushal-rathore-metacube**
https://spryker.atlassian.net/browse/CC-21838

 all available stores are displayed in the grid

### CODE REVIEW

File: resources/environments/environments_b2c.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
         "yves_second_user_password": "change123",
 
         "zed_admin_email": "admin@spryker.com",
-        "zed_admin_password": "change123"
+        "zed_admin_password": "change123",
+        "store_AT":"AT",
+        "store_DE":"DE"
```
Comment (AntonShopin) [inline resources/environments/environments_b2c.json:23]: it's a name or store id?

File: resources/pages/zed/zed_administration_page.robot [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*** Variables ***
+${administration_store_id_1_locator}    xpath=//tr[@class='odd']//td[contains(@class,'column-id_store')]
+${administration_store_id_2_locator}    xpath=//tr[@class='even']//td[contains(@class,'column-id_store')]
```
Comment (AntonShopin) [inline resources/pages/zed/zed_administration_page.robot:3]: * such a page doesn't exist. We have a 'Stores' page but not 'Administration'
* what if we'll have 10 stores. Do we need to create locators for them as they are dynamic? It's just a common Zed table, I don't see a need to describe exactly this specific table

File: resources/steps/zed_administration_steps.robot [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Resource    ../common/common.robot
+Resource    ../../resources/common/common_yves.robot
+Resource    ../../resources/pages/zed/zed_administration_page.robot
+
+*** Keywords ***
+
+Zed: check the avaialbe Stores
+    [Arguments]    ${store_id_1}    ${store_id_2}
+    Element Text Should Be    ${administration_store_id_1_locator}    AT
+    Element Text Should Be    ${administration_store_id_2_locator}    DE
```
Comment (AntonShopin) [inline resources/steps/zed_administration_steps.robot:11]: * You hardcoded expected values
* you use specific locators for 2 stores, don't do this. Stores are dynamic, reuse `Zed: table should contain:` or `Zed: table should contain non-searchable value:` with arguments in this keyword

File: tests/smoke/smoke_b2c.robot [hunk 4]
```diff
@@ -599,3 +600,8 @@ Refunds
     [Teardown]    Run keywords    Zed: login on Zed with provided credentials:    ${zed_admin_email}
     ...    AND    Zed: activate following discounts from Overview page:    Tu & Wed $5 off 5 or more    10% off $100+    20% off cameras    Tu & Wed €5 off 5 or more    10% off minimum order
  
+Display_stores_in_zed
+    Zed: login on Zed with provided credentials:    ${zed_admin_email}
+    Zed: go to first navigation item level:    Administration
+    Zed: go to second navigation item level:    Administration    Stores
```
Comment (AntonShopin) [inline tests/smoke/smoke_b2c.robot:606]: why do first you try to go to Administration (not existing) page and in the next step you go in the right way?

**Review by AntonShopin — CHANGES_REQUESTED** [review]

File: resources/steps/zed_administration_steps.robot (hunk 3 above)
Comment (khushal-rathore-metacube) [reply resources/steps/zed_administration_steps.robot:11]: Done

File: tests/smoke/smoke_b2c.robot (hunk 4 above)
Comment (khushal-rathore-metacube) [reply tests/smoke/smoke_b2c.robot:606]: Done..fixed

File: resources/environments/environments_b2c.json (hunk 1 above)
Comment (khushal-rathore-metacube) [reply resources/environments/environments_b2c.json:23]: fixed

File: resources/pages/zed/zed_administration_page.robot (hunk 2 above)
Comment (khushal-rathore-metacube) [reply resources/pages/zed/zed_administration_page.robot:3]: fixed

File: resources/steps/zed_administration_steps.robot [hunk 5]
```diff
@@ -0,0 +1,11 @@
+*** Settings ***
+Resource    ../common/common.robot
+Resource    ../../resources/common/common_zed.robot
+
+*** Keywords ***
+
+Zed: check the avaialbe Stores:
```
Comment (AntonShopin) [inline resources/steps/zed_administration_steps.robot:7]: typo in `avaialbe`

File: resources/environments/environments_b2c.json [hunk 6]
```diff
@@ -18,7 +18,9 @@
         "yves_second_user_password": "change123",
 
         "zed_admin_email": "admin@spryker.com",
-        "zed_admin_password": "change123"
+        "zed_admin_password": "change123",
+        "store_name_1":"AT",
+        "store_name_2":"DE"
```
[the remark this answers is not in the document]
Comment (khushal-rathore-metacube) [reply resources/environments/environments_b2c.json:23]: Fixed

File: resources/steps/zed_administration_steps.robot (hunk 5 above)
Comment (khushal-rathore-metacube) [reply resources/steps/zed_administration_steps.robot:7]: Fixed

**Review by AntonShopin — APPROVED** [review]

**Comment by khushal-rathore-metacube** [comment]
Conflicts has been resolved

**Pull request closed without merging**